.github/workflows/scale-test-100-gce.yaml

```diff
@@ -52,22 +57,38 @@ jobs:
       - name: Set Environment Variables
         uses: ./.github/actions/set-env-variables
 
-      - name: Get Cilium's default values
-        id: default_vars
-        uses: ./.github/actions/helm-default
-        with:
-          image-tag: ${{ github.sha }}
-
       - name: Set up job variables
         id: vars
         run: |
-          SHA="${{ github.sha }}"
+          if [ "${{ github.event_name }}" = "workflow_dispatch" ] || [ ${{ github.event.pull_request }} ] ; then
```

Comment:
`inputs` is only available when the workflow is triggered with `workflow_dispatch` and this workflow does not have the `workflow_dispatch` nor the `pull_request` directives. I had a quick look and there are a couple of other such checks in the repo, but it is a problem to fix another day.

This check is not required since this workflow does not run with `workflow_dispatch`.

Reply:
I would leave it if you don't mind. We are planning to make it possible to run on PRs with manual trigger if someone wants and on schedule for release-branches, which would require workflow_dispatch for ariane.

Reply:
Yes, certainly you can leave it for `workflow_dispatch`, it does nothing unless triggered with it.
But `github.event.pull_request` is wrong to be used this way, that part needs to be removed.

Reply:
Makes sense, thanks. Fixed.